mutable_tree.go

```diff
@@ -438,7 +447,7 @@ func (tree *MutableTree) Rollback() {
 func (tree *MutableTree) GetVersioned(key []byte, version int64) (
 	index int64, value []byte,
 ) {
-	if tree.versions[version] {
+	if !tree.allLoaded || tree.versions[version] {
```

Comment:
My intention is to fix this to be able to get the version by access of db even when we do `LazyLoadVersion`.
If we have done `LazyLoadVersion` then `tree.allLoaded` would be false.

Reply:
Ah, so this is optimization - we don't want to got through DB if we already used `LoadVersion` and the version was not there, right?

Reply:
I see, we want to try to load if we didn't use `LoadVersion` before.

Reply:
If a node started with `LazyLoadVersion`, what should this function do? Should it be able to give us only the last version that it loaded? Or should it access the DB and give any version to us if it's there?

Reply:
Exactly as you wrote - we should try to load it from DB. Maybe we could even do a small optimization to avoid loading if we already tried it:

```
exists, checked := tree.versions[version]
if (!tree.allLoaded && !checked) || exists {
    ...
```

This way, we avoid entering the load, if we already tried to do it and the he version didn't exist.

Reply:
That's a good point. I applied the checking to `VersionExists` and made other functions use it.